test/module.js

```diff
@@ -593,7 +592,10 @@ const cases = {
   },
 
   'placeholder-transparent': {
-    '::placeholder': { color: theme.colors['transparent'] },
+    '::placeholder': {
+      '--ow-placeholder-opacity': 1,
+      color: 'rgba(0,0,0,var(--ow-placeholder-opacity))',
+    },
```

Comment:
So this test case I think brings up a problem in how opacity is handled. 

I don't think we possibly can support alpha via colors. Once this PR lands there will be an alternative in the form of `placeholder-opacity-x`, but the `transparent` keyword may be a bit of a pain to work around.

Reply:
Just for clarification, `transparent`, if used in the theme, as it was, would cause issues due to the hex-only support, so this is with it's value instead being `#0000`, or transparent black.

Reply:
Ahh yeh.. good catch. Well if `#0000` works (it is the same solution as I would have gone for) then I have no problem with this for now!